aws/linked-account/role.tf

```diff
@@ -46,14 +51,21 @@ data "aws_iam_policy_document" "ternary_cmp_linked_account_agent_permissions" {
       "elasticmapreduce:ListClusters",
       "elasticmapreduce:ListInstances",
       "es:DescribeReservedInstances",
+      "lambda:ListFunctions",
+      "lambda:ListProvisionedConcurrencyConfigs",
       "memorydb:DescribeReservedNodes",
+      "organizations:ListAccounts",
```

Comment:
Not sure if this would cause a problem, per se, but it seems weird to include for linked accounts when it allows us to retrieve the info for _all_ accounts in the organization.

Reply:
Lgtm! I added to the TernaryCMPLinkedAccountAgent role in one of the dev linked accounts and it didn't cause any problems.

Reply:
🚀